include/adaflo/sharp_interface_util.h

```diff
@@ -1063,7 +1078,8 @@ compute_force_vector_regularized(const MatrixFree<dim, double> &matrix_free,
                                  const unsigned int             dof_index_ls,
                                  const unsigned int             dof_index_curvature,
                                  const unsigned int             dof_index_normal,
-                                 const unsigned int             quad_index)
+                                 const unsigned int             quad_index,
+                                 const double surface_tension_coefficient)
```

Comment:
It seems the order of parameters in this function is different from the other `compute_force_vector...` functions. Can you make the order consistent? To be more specific, it seems we should place `surface_tension_coefficient` before `curvature_solution` here. (And maybe also fix the order of the vectors.) Right now, it seems pretty confusing how to call the functions.

Reply:
Done!